src/main/java/org/jitsi/videobridge/simulcast/SimulcastSenderManager.java

```diff
@@ -144,11 +151,31 @@ private synchronized SimulcastSender getOrCreateSimulcastSender(
 
         if (simulcastSender == null) // Create a new sender.
         {
-            // Create a new sender.
-            int targetOrder = simulcastEngine.
-                    getVideoChannel().
-                    getReceiveSimulcastLayer();
+            int targetOrder = simulcastEngine
+                .getVideoChannel() // never null.
+                .getReceiveSimulcastLayer();
 
+            Endpoint sendingEndpoint = simulcastReceiver // never null.
+                .getSimulcastEngine() // never null.
+                .getVideoChannel() // never null.
+                .getEndpoint(); // might be null.
+
+            Endpoint receivingEndpoint = simulcastEngine // never null.
+                .getVideoChannel() // never null.
+                .getEndpoint(); // might be null.
```

Comment:
(1) Could I please ask you to rewrite that without repeating field accesses, methods calls and comments? I believe the source code will become more readable. For example,

VideoChannel videoChannel = simulcastEngine.getVideoChannel();
int targetOrder = videoChannel.getReceiveSimulcastLayer();
...
Endpoint receivingEndpoint = videoChannel.getEndpoint();

(2) I find these comments about "never null" and "might be null" distracting and unnecessary at times. For example, you check for receivingEndpoing right afterwards so it's pointless to me to say here that it might be null because your source code says it.

Additionally, with the repeating field accesses and method calls, you get into inconsistencies like the first simulcastEngine access doesn't have a comment but later on it has a comment.